applications-code-snippets/python-admission-system-models/readme.md

```diff
@@ -0,0 +1,21 @@
+## About Admission System Models
+
+Todo from @[PollenLight](https://github.com/PollenLight) Bhai if he's available 😛
```

Comment:
Hello @PollenLight Bhai👋
Interested in Hacktoberfest this year? You can start by making a PR to update this README. That'll be your first PR, then you can do 3 more as per your wish.
Thanks!